extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -274,6 +274,13 @@ public static enum Source {
             @ConfigItem
             public Provider secretProvider = new Provider();
 
+            /**
+             * The string representation of the private key content no matter including
+             * or excluding the ----- PRIVATE KEY / CERTIFICATE ----- prefix and postfix.
```

Comment:
```suggestion
             * The 'PRIVATE KEY' delimiters are not required.
```

Reply:
Certificate one would contain the public key...

Reply:
@IvanPuntev Actually, can you please drop the whole line, as inlined JWK format is also supported with that function call